Chapter-2-modules-separation/Src/docker-compose.yml

```diff
@@ -13,9 +13,9 @@ services:
 
   postgres:
     image: postgres:14.3
-    container_name: postgres
+    container_name: postgres_fitnet_2
```

Comment:
What do you think about

```
postgres_chapter_two
```

and in chapter 1:

```
postgres_chapter_one
```

Reply:
i restored file, sorry i thought that i restored this file before mark as ready.